pkg/kubectl/cmd/version.go

```diff
@@ -17,16 +17,29 @@ limitations under the License.
 package cmd
 
 import (
+	"errors"
 	"fmt"
-	"io"
-
 	"github.com/spf13/cobra"
+	"io"
 
+	"encoding/json"
+	"github.com/ghodss/yaml"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/version"
 )
 
+// We include both struct objects because json marshal does not handle the idea
+// of empty struct objects well and will include and empty server object in results
+type ClientVersionObj struct {
+	ClientVersion version.Info `json:"Client Version" yaml:"Client Version"`
+}
+
+type ClientAndServerVersionObj struct {
+	ClientVersion version.Info `json:"Client Version" yaml:"Client Version"`
```

Comment:
english language keys is odd here... I'd expect something like `client` and `server`